assets/src/block-editor/components/amp-preview.js

```diff
@@ -236,25 +236,23 @@ class AMPPreview extends Component {
 
 		return (
 			isEnabled && ! errorMessages.length && ! isStandardMode && (
-				<Tooltip text={ __( 'Preview AMP', 'amp' ) } >
-					<Button
-						className="amp-editor-post-preview"
-						href={ href }
-						label={ __( 'Preview AMP', 'amp' ) }
-						isSecondary
-						disabled={ ! isSaveable }
-						onClick={ this.openPreviewWindow }
-						ref={ this.buttonRef }
-					>
-						{ ampFilledIcon( { viewBox: '0 0 62 62', width: 18, height: 18 } ) }
-						<span className="screen-reader-text">
-							{
-								/* translators: accessibility text */
-								__( '(opens in a new tab)', 'amp' )
-							}
-						</span>
-					</Button>
-				</Tooltip>
+				<Button
+					className="amp-editor-post-preview"
+					href={ href }
+					title={ __( 'Preview AMP', 'amp' ) }
```

Comment:
The `label` attribute is changed to `title`. But other than that, this diff is just from indentation. There's no other change to `<Button>`